lib/ember-orbit/cache.js

```diff
@@ -0,0 +1,49 @@
+const get = Ember.get;
+
+import LiveQuery from 'ember-orbit/live-query';
+import { parseIdentifier } from 'orbit-common/lib/identifiers';
+
+export default Ember.Object.extend({
+  _orbitCache: null,
+  _identityMap: null,
+
+  init(...args) {
+    this._super.apply(this, ...args);
+
+    Ember.assert(get(this, '_orbitCache'), '_orbitCache is required');
+    Ember.assert(get(this, '_identityMap'), '_identityMap is required');
+  },
+
+  retrieve(path) {
+    return get(this, '_orbitCache').get(path);
+  },
+
+  retrieveRecord(type, id) {
+    return this.get('_identityMap').lookup({type, id});
+  },
+
+  retrieveAttribute(record, attribute) {
```

Comment:
I'm wondering if we should remove this from the Cache interface and move it to `model.retrieveAttribute` or even `attr.get` (attr.js).